demisto_sdk/commands/common/logger.py

```diff
@@ -1,142 +1,205 @@
 
-    l.propagate = False
+        def format(self, record):
+            message = logging.Formatter.format(self, record)
+            message = self.replace_escapes(message)
+            return message
 
-    return l
+        def replace_escapes(self, message):
+            for key in escapes:
+                message = message.replace(key, "")
+            return message
 
+    file_formatter = NoColorFileFormatter()
+    file_handler.setFormatter(fmt=file_formatter)
 
-logger: logging.Logger = logging_setup(verbose=1, quiet=False)
+    logging.basicConfig(
+        handlers=[console_handler, file_handler],
+        level=min(console_handler.level, file_handler.level),
+    )
 
+    root_logger: logging.Logger = logging.getLogger("")
+    set_demisto_handlers_to_logger(root_logger, console_handler, file_handler)
 
-def set_console_stream_handler(
-    logger: logging.Logger, handler: RichHandler = RichHandler(rich_tracebacks=True)
-):
-    """Set the console stream handler.
+    demisto_logger: logging.Logger = logging.getLogger("demisto-sdk")
+    set_demisto_handlers_to_logger(demisto_logger, console_handler, file_handler)
+    set_propagate(demisto_logger, False)
 
-    Args:
-        logger (logging.Logger): Logger.
-        handler (RichHandler, optional): RichHandler. Defaults to RichHandler(rich_tracebacks=True).
-    """
-    console_handler_index = -1
-    for i, h in enumerate(logger.handlers):
-        if h.name == "console-handler":
-            console_handler_index = i
-    if console_handler_index != -1:
-        logger.handlers[console_handler_index] = handler
-    else:
-        logger.addHandler(handler)
+    return demisto_logger
 
 
-def setup_rich_logging(verbosity: int, quiet: bool, log_path: Path, log_file_name: str):
-    """Override the default StreamHandler with the RichHandler.
+def set_demisto_handlers_to_logger(
+    logger: logging.Logger, console_handler, file_handler
+):
+    while logger.handlers:
+        logger.removeHandler(logger.handlers[0])
+    logger.addHandler(console_handler)
+    logger.addHandler(file_handler)
+    logger.level = min(console_handler.level, file_handler.level)
 
-    Setup logging and then override the default StreamHandler with the RichHandler.
 
-    Args:
-        verbosity (int): The log level to output.
-        quiet (bool): If True, no logs will be output.
-        log_path (Path): Path to the directory where the log file will be created.
-        log_file_name (str): The filename of the log file.
-    """
-    logger = logging_setup(
-        verbose=verbosity,
-        quiet=quiet,
-        log_path=log_path,  # type: ignore[arg-type]
-        log_file_name=log_file_name,
-    )
-    rich_handler = RichHandler(rich_tracebacks=True)
-    rich_handler.name = "console-handler"
-    set_console_stream_handler(logger, rich_handler)
+def get_log_file():
+    return current_log_file_path
+
+
+def set_propagate(logger_to_update: logging.Logger, propagate: bool = False):
```

Comment:
Maybe it's better to add to the setup?

Reply:
Removed altogether: https://github.com/demisto/demisto-sdk/pull/2707/commits/7338d0382b1a5ca9f54afb3c743db1809be8ef87